thirdeye-server/pom.xml

```diff
@@ -92,6 +92,12 @@
       <artifactId>mysql</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>javax.ws.rs</groupId>
+      <artifactId>javax.ws.rs-api</artifactId>
+      <version>2.1</version>
```

Comment:
add the main dependency to the parent pom. and in the server pom, you don't need the version. this allows us to control all dependencies from the parent pom and is a general practice.